packages/transaction-controller/src/utils/gas.ts

```diff
@@ -92,11 +97,15 @@ export async function estimateGas({
     await getLatestBlock(ethQuery);
 
   const blockGasLimitBN = hexToBN(blockGasLimit);
-
-  const fallback = BNToHex(
-    fractionBN(blockGasLimitBN, GAS_ESTIMATE_FALLBACK_BLOCK_PERCENT, 100),
+  const { gasEstimateFallback, isFixedGas } = getGasEstimateFallback(
+    chainId,
+    messenger,
   );
 
+  const fallback = isFixedGas
+    ? (gasEstimateFallback as string)
+    : BNToHex(fractionBN(blockGasLimitBN, gasEstimateFallback, 100));
```

Comment:
Can we also `log` the values we're using?

Reply:
Done